modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -762,6 +763,154 @@ public void testAddObjectEntry() throws Exception {
 				_simpleDTOConverterContext, _objectDefinition1,
 				parentObjectEntry1.getId()));
 
+		// Aggregation field without filters based on Many to Many relationship
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				_adminUser.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				_objectDefinition1.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"manyToManyRelationshipName",
```

Comment:
@pedro-oliveira446 use `StringUtil.randomId()` here, because we don't need to get the name of the relationship.

when we define a random name it's easy to know this name isn't relevant for the test.

Thanks!